data/afaanoromoo/articles/c4g19kgl85ko.json

```diff
@@ -419,7 +419,32 @@
     "locators": {
       "optimoUrn": "urn:bbc:optimo:asset:c4g19kgl85ko"
     },
-    "summary": "",
+    "summary": {
+      "blocks": [
+        {
+          "model": {
+            "blocks": [
+              {
+                "model": {
+                  "blocks": [
+                    {
+                      "model": {
+                        "attributes": [],
+                        "text": ""
```

Comment:
Reviewed this locally, 

`view-source:http://localhost:7080/afaanoromoo/articles/c4g19kgl85ko` gives the description value from the seoHeadline, since the promo summary is empty, but when I replace this empty string with "Hello" I see it is visible in the description meta tag. 

Looks great. Just that proptype warning error, mentioned above.